migration-guides/v9.0.0.md

```diff
@@ -0,0 +1,22 @@
+# v9.0.0 Migration Guide
+
+This version contains the following breaking changes:
+
+1. Removed Recompose library
+
+Further explanation of each item is detailed below.
+
+---
+
+## 1. Removed Recompose library
+
+While this won't break any imported components (they have been updated to work without recompose), it will break any instance of directly importing `wrapDisplayName` from the `utils` folder.
+
+To resolve this, remove that import and replace
+```
+wrapDisplayName(<component>, 'stringWrapper')
+``` 
+with an interpolated string:
+```
+`stringWrapper(${<component>.name})`
```

Comment:
Hmm, this isn't an _exact_ 1:1 replacement of the util. The first property it checks is `displayName`, followed by `name` ([source](https://github.com/acdlite/recompose/blob/3db12ce7121a050b533476958ff3d66ded1c4bb8/src/packages/recompose/getDisplayName.js#LL10C3-L10C64)).

Are any of the usages in this library where `<component>.displayName` yields a different, truthy result?

Reply:
Thanks, yeah I definitely should have just gone straight to the source!  Would it be worth it to actually just pull in the whole `wrapDisplayName` as a local util? It's clearly not that complicated and then this might not actually be a breaking change as `wrapDisplayName` would still exist in `utils`.

Reply:
Yep, let's do it. Maybe you can steal their tests too 🤪